alternator/ttl.cc

```diff
@@ -436,9 +437,9 @@ class token_ranges_owned_by_this_shard {
     size_t _end_idx;
     std::optional<dht::selective_token_range_sharder> _intersecter;
 public:
-    token_ranges_owned_by_this_shard(replica::database& db, schema_ptr s)
+    token_ranges_owned_by_this_shard(data_dictionary::database db, schema_ptr s)
```

Comment:
Nitpick (not important): As you noted in the commit message, the replication map is not available in data_dictionary, and you need to use the "real_database()" . So why not just leave this function to take replica::database?

Reply:
I started with a wholesale change until I hit the snag. I can change it back.